app/src/shared/hooks/useBalance.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import { useState, useEffect, useCallback, useMemo } from 'react'
+
+import { getDataTokenBalance, getEthBalance } from '$mp/utils/web3'
+import type { Address } from '$shared/flowtype/web3-types'
+
+export const BalanceType = {
+    ETH: 'ETH',
+    DATA: 'DATA',
+}
+
+export async function getBalance(address: Address, balanceType: $Values<typeof BalanceType>, usePublicNode?: boolean = false) {
```

Comment:
Can we use named parameters instead of positional arguments?  It's particularly bad for boolean flags because there's no way to know what the parameter is about without looking up the signature.

i.e. 
```js
getBalance(address, balanceType, true)

// vs 

getBalance({
  address,
  balanceType,
  usePublicNode: true,
})

```

Reply:
Good idea, changed it 👍